packages/php-platform/.github/workflows/build-tagged-version.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+    push:
+        tags:
+            - '**'
+    workflow_dispatch:
+name: Build PHP Platform Docker image for released version
+jobs:
+    rebuild-php-fpm-images:
+        name: Build tag
+        runs-on: ubuntu-22.04
+        permissions:
+            contents: read
+        steps:
+            -   name: GIT checkout tag - ${{ github.ref }}
+                uses: actions/checkout@v3
+            -   name: Login to DockerHub
+                uses: docker/login-action@v2
+                with:
+                    username: ${{ secrets.DOCKERHUB_USERNAME }}
+                    password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
Why are you using DockerHub instead of GitHub Package Registry? Is it because you do not want to use ghcr.io in images names?

Reply:
I decided to use the default Docker registry for several reasons: 
- I still find it the "industry standard" and I find it easier to write the `docker run` or FROM instructions from the top of my head
- Dockerhub offers some additional nice tooling – for example, Docker Scout for image analysis
- I find the user interface on dockerhub more user-friendly